src/utils/is-logged-in.js

```diff
@@ -0,0 +1,16 @@
+
+import localStorage from 'localStorage';
+
```

Comment:
Would you please try documenting ``` isLoggedIn``` method

Reply:
Thanks for the feedback 
isLoggedIn method now